lib-rpc/ocamlformat_rpc_lib.ml

```diff
@@ -159,10 +165,171 @@ module V1 :
   end
 end
 
-type client = [`V1 of V1.Client.t]
+module V2 : sig
+  module Command :
+    Command_S
+      with type t =
+        [ `Halt
+        | `Unknown
+        | `Error of string
+        | `Path of string
+        | `Config of (string * string) list
```

Comment:
Now that we have a version bump, we might as well get rid of these commands now that they can be passed as option to the `format` command.

Reply:
They can be passed to the `format` command but in this case they are not persistent, so they cannot be "emulated" by the `format` command.
I'm OK to remove them, as I guess we always want to recompute the options from the `path`, in case the `.ocamlformat` file (or other relevant file/environment variable) have been modified.

Reply:
We can certainly cache the `.ocamlformat` lookups. I think most clients would unconditionally send the `path` command before every `format` anyway, the `path` command is not an optimization.

Reply:
I'm fine with removing `Config` and `Path` as well

Reply:
I'm fine with removing `Config` and `Path` as well